src/coreclr/inc/cvconst.h

```diff
@@ -3646,6 +3648,7 @@ typedef enum CV_HREG_e {
 
     // Note:  Next set of platform registers need to go into a new enum...
     // this one is above 44K now.
+    // TODO for LOONGARCH64: ?!?!?
```

Comment:
Hi, @jkotas 
Is it needed to be modified for LoongArch64 ?

Reply:
I think all changes in this file should be reverted.

This file has constants and enum for codeview symbol format. The codeview symbol format is only used on Windows. Unless you plan to work on porting Windows OS to LoongArch64, you should not need to worry about codeview support for LoongArch64.

Reply:
It seems that these are unused, so I will revert this file.